raiden-ts/src/transfers/epics/mediate.ts

```diff
@@ -34,34 +79,51 @@ function shouldMediate(action: transferSigned, address: Address, { caps }: Raide
  * @param deps - Raiden epic deps
  * @param deps.address - Our address
  * @param deps.config$ - Latest observable
+ * @param deps.log - Logger instance
  * @returns Observable of outbound transfer.request actions
  */
 export function transferMediateEpic(
   action$: Observable<RaidenAction>,
-  {}: Observable<RaidenState>,
-  { address, config$ }: RaidenEpicDeps,
+  state$: Observable<RaidenState>,
+  { address, config$, log }: RaidenEpicDeps,
 ) {
   return action$.pipe(
     filter(transferSigned.is),
-    withLatestFrom(config$),
+    withLatestFrom(config$, state$),
     filter(([action, config]) => shouldMediate(action, address, config)),
-    map(([{ payload: { message: transf }, meta: { secrethash } }]) =>
+    map(([action, config, state]) => {
+      const transferState = state.transfers[transferKey(action.meta)];
+      const locked = action.payload.message;
```

Comment:
There's only a single usage of `locked` in our codebase, which is a shorthand (although still descriptive enough) for `LockedTransfer`. I thought this would make it clearer, but I can rename it back to `message`, or maybe `receivedTransfer`, if that makes it clearer for you.

Reply:
Renamed back.